### ISSUE TITLE
New (Comfy-only) auto-segmentation-detailer! aka, "ADetailer"++

### ISSUE THREAD

**Issue by samliu315** [Stability-AI/StableSwarmUI#464]
Hi, I got an error when I tried to use this function: ComfyUI execution error: expected str, bytes or os.PathLike object, not NoneType.

My usage is to fill in the end of the prompt directly: <segment:girl's face>

Also, a photo of a woman standing in the street smiling <segment:yolo-face_yolov8m-seg_60.pt-1> a ​​beautiful woman's smiling face, **it works for me.**

What's wrong? Or can you give a more detailed tutorial?

**Comment by southbayjay** [comment]
This repo is way out of date. The new version of (now called) SwarmUI, is located here.. This version is no longer maintained. You need to install the new version. 

Also join the discord to get find the entire Swarm community.. https://discord.gg/XDGYCWBG

https://github.com/mcmonkeyprojects/SwarmUI

**Comment by samliu315** [comment]
Thank you. Actually I have already got the new version. Does the current version support this detailer?

**Comment by samliu315** [comment]
> This repo is way out of date. The new version of (now called) SwarmUI, is located here.. This version is no longer maintained. You need to install the new version.这个仓库已经过时了。新版本的（现在称为）SwarmUI，位于此处。该版本不再维护。您需要安装新版本。
> 
> Also join the discord to get find the entire Swarm community.. https://discord.gg/XDGYCWBG还可以加入不和谐来找到整个 Swarm 社区.. https://discord.gg/XDGYCWBG
> 
> https://github.com/mcmonkeyprojects/SwarmUI

**Thank you. Actually I have already got the new version. Does the current version support this detailer?**

**Comment by southbayjay** [comment]
Yes, there's a few ways you can do it but the most practical (imo) is with segmentation, type out your prompt as normal and then add something like this.. 

`<segment:face> a beautiful hd photo of a woman's face`

or as you did it it like this

`<segment:yolo-face_yolov8n-seg2_60.pt> a ​​beautiful woman's smiling face`

**Comment by southbayjay** [comment]
@samliu315 let me know if you need anymore help. I’m signing out for the evening but your welcome to swing by the Discord and post there. Tag me if you do @southbayjay
